src/libsystemd/sd-journal/test-journal-send.c

```diff
@@ -90,6 +90,8 @@ static void test_journal_send(void) {
         assert_se(sd_journal_sendv(graph2, 1) == 0);
         assert_se(sd_journal_sendv(message1, 1) == 0);
         assert_se(sd_journal_sendv(message2, 1) == 0);
+
+        closelog();
```

Comment:
maybe add a comment why this is added here, it's otherwise hard to see why we bother, given the fd is not really managed by use, but by libc

Reply:
Sure. A brief comment is added.